client/components/dataviews/style.scss

```diff
@@ -86,6 +86,7 @@
 			text-transform: uppercase;
 		}
 
+		span.dataviews-view-table-header,
```

Comment:
This overrides the rule above, 

```
color: var( --color-accent-80 );
```

Do we need to set this color at all?

Reply:
I think you're right, and for `button` [it's being set](https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/dataviews-layouts/table/style.scss#L146-L148) to `#1e1e1e` anyways. Removed the color rule and updated in https://github.com/Automattic/wp-calypso/commit/26b67ddae3908b73541b32a7fd327c04025ac57e.